js/libs/keycloak-admin-client/src/defs/componentTypeRepresentation.ts

```diff
@@ -5,6 +5,8 @@ import type { ConfigPropertyRepresentation } from "./configPropertyRepresentatio
  */
 export default interface ComponentTypeRepresentation {
   id: string;
+  displayType: string;
```

Comment:
@edewit Does it make sense to you? Now we resolve both data from the payload returned from the new endpoint (no need to query server info state).

Reply:
Yeah, that makes sense, makes it a lot easier